code-challenges.test.js

```diff
@@ -14,43 +14,100 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibonacciLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+// describe('fibonacciSeq', () => {
+//   it('takes in a number (greater than 2) and returns an array containing the Fibonacci sequence', () => {
+// const fibonacciLength1 = 6
+//     // Expected output: [1, 1, 2, 3, 5, 8]
+// const fibonacciLength2 = 10
+//     // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+//     expect(fibonacciSeq(fibonacciLength1)).toEqual([1, 1, 2, 3, 5, 8])
+//     expect(fibonacciSeq(fibonacciLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
 
-const fibonacciLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+//   })
+// })
+// --- ReferenceError: fibonacciSeq is not defined ---
 
 // b) Create the function that makes the test pass.
-
 // Pseudo code:
 
+// function fibonacciSeq(length) {
+//   // fibonacci seq must be greater than 2, just based off its principal. so inform user that it will be invalid 
+//   if (length <= 2) {
+//     return ('Invalid input.')
+//   }
+//   // initial sequence greater than 2, which then calculates the next "fibonacci" number until it reaches the specified length
+//   const sequence = [1, 1]
+//   for (let i = 2; i < length; i++) {
+//     // calculate the next number by adding the last two numbers in the sequence
+//     const nextNumber = sequence[i - 1] + sequence[i - 2]
+//     // add the last number to the new sequence
+//     sequence.push(nextNumber)
+//   }
+//   return sequence
+// }
+// // console log each given sequence passed through the above function
+// const one = [1]
+// console.log(fibonacciSeq(one))
+// // Output -> Invalid input.
+// console.log(fibonacciSeq(fibonacciLength1))
+// // Output -> [1, 1, 2, 3, 5, 8]
+// console.log(fibonacciSeq(fibonacciLength2))
+// // Output -> [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
```

Comment:
Excellent work, Zenia.  I'm curious why you used the function keyword instead of ES6 arrow syntax.  Please make sure you are using the syntax we teach in class.  You're doing great - keep up the good work!